SingularityUI/app/components/dashboard/DashboardPage.jsx

```diff
@@ -13,6 +14,7 @@ const DashboardPage = () => (
   <div>
     <Header />
     <MyRequests />
+    <MyGroupRequests group={'hs-salesproduct'} />
```

Comment:
Do we want to pass in a constant hubspot-specific value to this prop?

Also, if so this should be `group="hs-salesproduct"`.